Modules/Quotas/test/Quotas.Domain.Tests/QuotaIdTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Domain.Tests;
+public class QuotaIdTests
+{
+    [Fact]
+    public void Can_create_quota_id_with_valid_value()
+    {
+        var validQuotaIdPrefix = "QUO";
+        var validIdLengthWithoutPrefix = 17;
+        var validIdValue = validQuotaIdPrefix + TestDataGenerator.GenerateString(validIdLengthWithoutPrefix);
+
+        var quotaId = QuotaId.Create(validIdValue);
```

Comment:
`quotaId` is a misleading name, because it actually contains a Result, and not the id